TheMovieDatabase/ViewControllers/FullPictureScroll/View/FullPictureScrollViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  FullPictureScrollViewController.swift
+//  TheMovieDatabase
+//
+//  Created by Evgeny Syrtsov on 5/1/20.
+//  Copyright © 2020 Evgeny Syrtsov. All rights reserved.
+//
+
+import UIKit
+
+final class FullPictureScrollViewController: UIViewController {
```

Comment:
I don't like `FullPicture` name :(
What does it mean? Full picture? How does a picture maybe full or not full? 

Also, I haven't heard anything about `ScrollViewController` can you shed some light on this?

Reply:
My suggestions: `ImageViewerViewController`, `GalleryViewController` (first one is better for me)

Reply:
Note: it's not okay to leak implementation details through a name. There you leaked that you implemented your controller with a scroll view. Maybe it's okay when you have couple implementations, but it's not your case